install_config/install/docker_registry.adoc

```diff
@@ -1049,6 +1062,61 @@ middleware:
 <3> A configuration option that can be overridden by the environment variable `*REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_PROJECTCACHETTL*`, specifying an eviction timeout for project quota objects. It takes a valid time duration string (for example, *2m*). If empty, you get the default timeout. If zero (*0m*), caching is disabled.
 ====
 
+[[middleware-repository-pullthrough]]
+===== Image Pullthrough
+
+If enabled, the registry will attempt to fetch requested blob from a remote
+registry unless the blob exists locally. The remote candidates are calculated
+from **DockerImage** entries stored in status of the
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#image-streams[image
+stream], a client pulls from. All the unique remote registriy references in
+such entries will be tried in turn until the blob is found. The blob, served
+this way, will not be stored in the registry.
+
+This feature is on by default. However, it can be disabled using a
+xref:docker-registry-configuration-reference-middleware[configuration option].
+
+[[middleware-repository-acceptschema2]]
+===== Manifest schema v2 support
+// is there a better place for this?
```

Comment:
It seems to me like it clutters the configuration section. Perhaps it could be moved somewhere into architecture/images.